test/output/anon-function/expected.mjs

```diff
@@ -0,0 +1 @@
+_.e([["default",()=>default]]);function default() {}
```

Comment:
Instead of giving the anonymous function a name, I think you can just embed it inside the `_.e(...)` expression, like so:
```js
_.e([["default",(x=>()=>x)(function () {})]]);
```
The `x=>()=>x` bit is just an arrow function that takes a value and returns a function that always returns that value.

Reply:
I think is is better as a declaration for two reasons:
-  Currently `@std/esm` does some work for keeping the original line numbers ([relevant test](https://github.com/standard-things/esm/tree/master/test/output/lines)). If anonymous declarations get moved to the top, it would be impossible.
- The `default` function name is more development-friendly when checking error stack traces (in a native implementation, their name would be `*default*`)

Reply:
Good points!